proto/substrait/algebra.proto

```diff
@@ -859,6 +857,13 @@ message Expression {
       int32 scale = 3;
     }
 
+    message PrecisionTimestamp {
+      // Sub-second precision, 0 means the value given is in seconds, 3 is milliseconds, 6 microseconds, 9 is nanoseconds
+      int32 precision = 1;
+      // Time passed since 1970-01-01 00:00:00.000000 in UTC for PrecisionTimestampTZ and unspecified timezone for PrecisionTimestamp
+      int64 value = 2;
```

Comment:
```suggestion
      uint32 precision = 1;
      // Time passed since 1970-01-01 00:00:00.000000 in UTC for PrecisionTimestampTZ and unspecified timezone for PrecisionTimestamp
      uint64 value = 2;
```

Reply:
Thanks, I fixed the value to be uint64. 

The precision is int32 in the [type](https://github.com/substrait-io/substrait/pull/659/files#diff-65090cba5b925bfa9e08ee1ef22dcef8f347e99da3165b73f5f25e0008de521eR169) so I think it's better to keep it as such?

Reply:
Agreed. Keeping precision as int32 seems fine.